packages/client/hmi-client/src/components/intervention-policy/tera-intervention-summary-card.vue

```diff
@@ -16,25 +16,33 @@
 			<li v-for="(dynamicIntervention, index) in intervention.dynamicInterventions" :key="`dynamic-${index}`">
 				Set {{ dynamicIntervention.type }} <strong>{{ dynamicIntervention.appliedTo }}</strong> to
 				<strong>{{ dynamicIntervention.value }}</strong> when
-				<strong>{{ dynamicIntervention.parameter }}</strong> crosses the threshold&nbsp;<strong>{{
-					dynamicIntervention.threshold
-				}}</strong
+				<strong>{{ dynamicIntervention.parameter }}</strong> crosses the threshold&nbsp;<strong
+					>{{ dynamicIntervention.threshold }} {{ getUnit(dynamicIntervention) }}</strong
 				>.
 			</li>
 		</ul>
 	</div>
 </template>
 
 <script setup lang="ts">
-import { Intervention } from '@/types/Types';
+import { Intervention, DynamicIntervention } from '@/types/Types';
 import { CalendarSettings, getTimePointString } from '@/utils/date';
 import { isEmpty } from 'lodash';
 
 const props = defineProps<{
 	intervention: Intervention;
+	modelUnits?: any;
```

Comment:
If possible, let's avoid using `any` as it makes it harder to known how to use this component.